libs/design-system/src/lib/Components/InternalCodeblock/index.tsx

```diff
@@ -104,29 +103,6 @@ export function InternalCodeblock({
           )}
         />
       </div>
-      <Dialog
```

Comment:
Was this removed intentionally?

Reply:
yeah consider this whole thing an experiment that didn't work out. removing this changed the way it failed slightly: without dialog the page will load and errors are logged in the browser console, with the dialog the page doesn't load at all.